PolicyServerElasticsearchContainer/pom.xml

```diff
@@ -418,49 +410,26 @@
                             <alias>policy-server</alias>
                             <name>${container.policyserver.name}</name>
                             <build>
-                                <from>${container.policyelasticsearchbase.name}</from>
+                                <from>${container.policyelasticsearchtomcatbase.name}</from>
                                 <labels>
                                     <Build.Number>${project.version}</Build.Number>
                                     <Build.Date>${maven.build.timestamp}</Build.Date>
                                     <Git.Branch>${git.branch}</Git.Branch>
                                     <Git.Commit>${git.revision}</Git.Commit>
                                 </labels>
-                                <!-- Expose port for tomcat -->
```

Comment:
We shouldn't be removing this `EXPOSE` unless we also put it in the base image.

Reply:
These ports are already exposed in the tomcat base image:
https://github.com/CAFapi/opensuse-tomcat-image/blob/develop/src/main/docker/Dockerfile#L63

Reply:
My mistake, the ports are exposed in the tomcat base container which is then thrown away, I have added the ports to be exposed back into the container pom.